src/analysermodel.cpp

```diff
@@ -16,10 +16,39 @@ limitations under the License.
 
 #include "libcellml/analysermodel.h"
 
+#include <algorithm>
+
+#include "libcellml/variable.h"
+
 #include "analysermodel_p.h"
 
 namespace libcellml {
 
+bool AnalyserModel::AnalyserModelImpl::haveEquivalentVariables(const Variable *variable1,
```

Comment:
why is this "have" ? is it not checking that variable1 and variable2 are equivalent?

Reply:
Yes, renamed to `areEquivalentVariables()` in commit c168cc9a1.